Gems/Terrain/Code/Source/Components/TerrainWorldRendererComponent.cpp

```diff
@@ -85,15 +99,65 @@ namespace Terrain
+                    ->DataElement(AZ::Edit::UIHandlers::Slider, &ClipmapConfiguration::m_macroClipmapMaxResolution, "Macro clipmap max resolution", "The resolution of the highest resolution clipmap in the stack.")
+                        ->Attribute(AZ::Edit::Attributes::Min, 2.0f)
+                        ->Attribute(AZ::Edit::Attributes::SoftMin, 2.0f)
+                        ->Attribute(AZ::Edit::Attributes::Max, 10.0f)
+                        ->Attribute(AZ::Edit::Attributes::SoftMax, 10.0f)
+                    ->DataElement(AZ::Edit::UIHandlers::Slider, &ClipmapConfiguration::m_detailClipmapMaxResolution, "Detail clipmap max resolution", "The resolution of the highest resolution clipmap in the stack.")
+                        ->Attribute(AZ::Edit::Attributes::Min, 512.0f)
+                        ->Attribute(AZ::Edit::Attributes::SoftMin, 512.0f)
+                        ->Attribute(AZ::Edit::Attributes::Max, 2048.0f)
+                        ->Attribute(AZ::Edit::Attributes::SoftMax, 2048.0f)
+                    ->DataElement(AZ::Edit::UIHandlers::Slider, &ClipmapConfiguration::m_macroClipmapScaleBase, "Macro clipmap scale base", "The scale base between two adjacent clipmap layers.")
+                        ->Attribute(AZ::Edit::Attributes::Min, 2.0f)
+                        ->Attribute(AZ::Edit::Attributes::SoftMin, 2.0f)
+                        ->Attribute(AZ::Edit::Attributes::Max, 4.0f)
+                        ->Attribute(AZ::Edit::Attributes::SoftMax, 4.0f)
+                    ->DataElement(AZ::Edit::UIHandlers::Slider, &ClipmapConfiguration::m_detailClipmapScaleBase, "Detail clipmap scale base", "The scale base between two adjacent clipmap layers.")
+                        ->Attribute(AZ::Edit::Attributes::Min, 2.0f)
+                        ->Attribute(AZ::Edit::Attributes::SoftMin, 2.0f)
+                        ->Attribute(AZ::Edit::Attributes::Max, 4.0f)
+                        ->Attribute(AZ::Edit::Attributes::SoftMax, 4.0f)
+                    ->DataElement(AZ::Edit::UIHandlers::Slider, &ClipmapConfiguration::m_macroClipmapMarginSize, "Macro clipmap margin size", "The margin of the clipmap where the data won't be used.")
+                        ->Attribute(AZ::Edit::Attributes::Min, 1u)
+                        ->Attribute(AZ::Edit::Attributes::SoftMin, 1u)
+                        ->Attribute(AZ::Edit::Attributes::Max, 8u)
+                        ->Attribute(AZ::Edit::Attributes::SoftMax, 8u)
+                    ->DataElement(AZ::Edit::UIHandlers::Slider, &ClipmapConfiguration::m_detailClipmapMarginSize, "Detail clipmap margin size", "The margin of the clipmap where the data won't be used.")
+                        ->Attribute(AZ::Edit::Attributes::Min, 1u)
+                        ->Attribute(AZ::Edit::Attributes::SoftMin, 1u)
+                        ->Attribute(AZ::Edit::Attributes::Max, 8u)
+                        ->Attribute(AZ::Edit::Attributes::SoftMax, 8u)
```

Comment:
When would a user need to adjust these values? What benefit does a smaller or larger margin provide?

Reply:
This one is for clipmap updating frequency. Higher bound size means more data in the clipmap will be not used, but it will trigger clipmap update less frequently.